Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -1078,19 +1146,20 @@ public void ComputeProjectBaseDir_BestCommonRoot_ProjectOutsideRoot_LogsWarning(
         }
 
         [TestMethod]
-        public void ComputeProjectBaseDir_NoBestCommonRoot_LogsFallbackWarning()
+        public void ComputeProjectBaseDir_NoBestCommonRoot_ReturnsNull()
         {
             var logger = new TestLogger();
-            var sut = new PropertiesFileGenerator(new AnalysisConfig { SonarOutputDir = @"Z:\Fallback" }, logger);
+            var sut = new PropertiesFileGenerator(new AnalysisConfig(), logger);
             var projectPaths = new[]
             {
                 new DirectoryInfo(@"C:\RootOnce"),
                 new DirectoryInfo(@"D:\AlsoOnce"),
                 new DirectoryInfo(@"E:\NotHelping"),
             };
 
-            sut.ComputeProjectBaseDir(projectPaths).FullName.Should().Be(@"Z:\Fallback");
```

Comment:
There is no longer a predefined fallback. If the common root cannot be found, we ask the user to provide the project dir parameter